### DISCUSSION TITLE
How to access the query logs

### DISCUSSION THREAD

**Discussion by duffysp** [AdguardTeam/AdGuardHome#5653] [Q&A] [answered]
Hoping this is a quick answer for someone.
I'm looking to see how to access the query logs from SSH. I have Adguard installed on a Pi and find the UI a bit limiting. I'm looking to export the logs to desktop so I can clean them up (remove "noise") and apply advanced filters that the UI doesnt allow. 
I've checked google, the github and the Q&A section and cant find anything saying how to access the logs - only a couple of feature requests asking for an Export to CSV function.
This makes me think its either super easy that no one has asked before or not possible. 

Anyone able to help?
Thanks.

**Comment by ainar-g** [comment, marked as answer]
Hello.  The query log is stored in files `data/querylog.json` (and, possibly, `data/querylog.json.1`) in AdGuard Home's working directory.  They contain a stream of JSON objects, the format of which is not stable, but you can still process them with e.g. `jq`.

**Reply by Quinferris** [reply]
Trying to see why I can't see the blocked queries? Is there a setting to enable a more verbose log file for querylog? I notice some are missing.

**Reply by KelvinUceta** [reply]
Hi,
In the _querylog.json_ there's a column named "IsFiltered", my guess is that those are the blocked one, please confirm!

**Comment by zero77** [comment]
Are there any open issues or plans to use something like sqlite instead
